js/site-orientation.js

```diff
@@ -9,6 +9,9 @@ var helpers = require('./helpers');
 var uri = window.location.toString();
 var uriQuery = helpers.sanitizeQueryParams(URI.parseQuery(window.location.search));
 
+// Use a fresh localstorage item once we're in production
+var STORAGE_ITEM = window.CANONICAL_BASE === 'https://fec.gov' ? 'FEC_BANNER_COLLAPSED_PROD' : 'FEC_BANNER_COLLAPSED';
```

Comment:
Can you break this into another line? For readability I'm hesitant on having more than one setter in one line.